package.json

```diff
@@ -8,13 +8,16 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "pretty-format": "^26.0.0",
-    "ts-jest": "^26.0.0"
+    "pretty-format": "26.x",
+    "ts-jest": "26.x"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.0",
-    "@types/node": "^14.0.4",
-    "jest": "^26.0.1",
+    "@types/jest": "26.x",
+    "@types/node": "12.x",
+    "cross-spawn": "7.x",
+    "fs-extra": "9.x",
+    "jest": "26.x",
+    "lodash.memoize": "4.x",
```

Comment:
Is there any advantage of `26.x` over `^26.0.0`? Usually only the `^`-syntax is used, also by tools like `npm update` or `npm upgrade` or dependabot.

Reply:
This syntax is shorter than the original syntax ^x.x.x which behaves the same as the original syntax, dependabot also understands this so that is assured